.gitignore

```diff
@@ -18,3 +18,9 @@ build.xml
 banmanagement/settings.php
 cache
 /target
+
+# Bash scripts #
+*.sh
+
+# Vim backup files #
+*~
```

Comment:
Not sure these are necessary? If you have these locally it would be better to add them to your global gitignore file instead of in this project

Reply:
Ok, didn't know about global gitignore, thanks